### PR TITLE
added (or Beatmap Appreciation Team) in BAT

### PR DESCRIPTION
**Pull request by apap04**
Added this for users to know what BAT is.

### CODE REVIEW

**Comment by MegaApplePi** [comment]
The meaning of that term is in the definition.

I don't think there is a need for this and besides, this will only create more confusing section links.

File: wiki/Glossary/en.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Bancho is the easily expandable server coded by peppy.
 Bancho replaces the need for a third-party IRC server in the management of all online content in osu!.
 Logging with an osu!account ingame will automatically attempt to connect to Bancho until done successfully.
 
-### BAT
+### BAT (or Beatmap Appreciation Team)
 
 **As of February 1, 2015, the BAT was renamed to _Beatmap Nomination Group_**
```
Comment (kj415j45) [inline wiki/Glossary/en.md]: why not add them to this line(L40)

Comment (MegaApplePi) [inline wiki/Glossary/en.md]: the meaning is placed under that line, but that is a notice to say that the team name was changed...

Comment (MegaApplePi) [inline wiki/Glossary/en.md]: if really, the I guess the meaning can be moved to that line, but that line isn't meant to be used as the definition

Comment (kj415j45) [inline wiki/Glossary/en.md]: if it's the first time that it appear in this article, it **can** be there(not **must**)

Comment (apap04) [inline wiki/Glossary/en.md]: Close???

Comment (kj415j45) [inline wiki/Glossary/en.md]: depends on you

i suggest close it
if you want to keep it, please also fix others like AFK, BPM, MAT, .....................

**Pull request closed without merging**